ogr/services/pagure.py

```diff
@@ -198,7 +202,7 @@ def __init__(
         self.namespace = namespace
 
     def __str__(self) -> str:
-        return f"namespace={self.namespace} repo={self.repo}"
+        return f"namespace=\"{self.namespace}\" repo=\"{self.repo}\""
```

Comment:
Please, be consistent and do it in a same way as you did in the case of Github.

```suggestion
        return f"GithubProject(namespace=\"{self.namespace}\", repo=\"{self.repo}\")"
```

Reply:
Makes sense. I will make the changes. Thanks for your valuable feedback !